docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -31,10 +31,14 @@
 
 \section{Problem Statement}
 
-\wss{You should check your problem statement with the
-\href{https://github.com/smiths/capTemplate/blob/main/docs/Checklists/ProbState-Checklist.pdf}
-{problem statement checklist}.}
-\wss{You can change the section headings, as long as you include the required information.}
+%\wss{You should check your problem statement with the
+%\href{https://github.com/smiths/capTemplate/blob/main/docs/Checklists/ProbState-Checklist.pdf}
+%{problem statement checklist}.}
+%\wss{You can change the section headings, as long as you include the required information.}
+
+This section will outline the Problem Statement the Grocery Spending Tracker is trying to solve. It will go 
```

Comment:
the problem statement describes the problem we are trying to solve, we aren't solving the statement

Reply:
Changed it from Problem Statement to just plain problem. Not sure if that's better. I can also just rewrite it if you still don't like it.